students/headieh_godwin/lesson05/mailroom.py

```diff
@@ -2,64 +2,41 @@
 import inquirer
 import os
 import time
-
-donor1 = {
-    "name": "Karen",
-    "donations": (20,20,100)
-}
-donor2 = {
-    "name": "Susan",
-    "donations": (20)
-}
-donor3 = {
-    "name": "Larry",
-    "donations": (40,50)
-}
-donor4 = {
-    "name": "Curly",
-    "donations": (20.99,20,100)
-}
-donor5 = {
-    "name": "Mo",
-    "donations": (2)
-}
+import sys
+import re
 
 donors = {
-  "donor1": donor1,
-  "donor2": donor2,
-  "donor3": donor3,
-  "donor4": donor4,
-  "donor5": donor5
+  "Karen": (20,20,100),
+  "Susan": (20,),
+  "Larry": (40,50),
+  "Curly": (20.99,20,100),
+  "Mo": (2,)
```

Comment:
using a tuple is incorrect here , tuples are immutable and are used for specific use cases

your data here is actually mutable and should be stored in a list.